yogstation/code/modules/shipbreaker/shipbreaker.dm

```diff
@@ -0,0 +1,28 @@
+/datum/map_template/shipbreaker
+	should_place_on_top = FALSE
```

Comment:
these should be placed on top so shipbreaking can be spawned on stuff like asteroid turfs..

Reply:
ok

Reply:
updated